api/inc/box_config.h

```diff
@@ -64,6 +65,7 @@ UVISOR_EXTERN const uint32_t __uvisor_mode;
         UVISOR_MIN_STACK(stack_size), \
         context_size, \
         __uvisor_box_namespace, \
+        NULL, \
```

Comment:
Nope. Only the main box can indicate the hooks.

I'll rework this to be part of uvisor_config instead. It doesn't make much sense for a box-specific configuration to do the system configuration.